elements/ABD-85148/initialize.js

```diff
@@ -403,9 +403,54 @@ function(instance, context) {
             wordWrapWidth: rect.width,
             breakWords: true,
         });
-        true ? console.log("label", label, rect, rect.getBounds()) : null;
         rect.addChild(label);
         label.position.set(10, 10);
+
+        let { width, height } = rect;
+        let x = 0, y = 0;
+
+
+        let labelX = 10;
+        let labelY = 10;
+        let labelWidth = label.width;
+        let labelHeight = label.height;
+
+
+        // if the recntangle is not highlighted, create the custom hit area that wraps around the rectangle and the label, else default to the visible area defined by pixis internals
+        if (!rect.isHighlighted) {
+            invisibleHitArea = new PIXI.Polygon([
+                //starts wrapping around the outer rectangle
+                new PIXI.Point(x - 5, y - 5), // top left corner of outer rectangle
+                new PIXI.Point(x + width + 5, y - 5), // top right corner of outer rectangle
+                new PIXI.Point(x + width + 5, y + height + 5), // bottom right corner of outer rectangle
+                new PIXI.Point(x - 5, y + height + 5), // bottom left corner of outer rectangle
+                new PIXI.Point(x - 5, y - 5), // top left corner of outer rectangle
+
+                // //starts wrapping around the label
+                new PIXI.Point(labelX, labelY), // top left corner of label
+                new PIXI.Point(labelX + labelWidth, labelY), // top right corner of label
+                new PIXI.Point(labelX + labelWidth, labelY + labelHeight), // bottom right corner of label
+                new PIXI.Point(x + 10, labelY + labelHeight), // bottom of label, 5px inside of outer rectangle
+
+                // //starts wrapping around the inner rectangle
+                new PIXI.Point(x + 10, y + height - 10), // bottom left corner of inner rectangle
+                new PIXI.Point(x + width - 10, y + height - 10), // bottom right corner of inner rectangle
+                new PIXI.Point(x + width - 10, y + 10), // top right corner of inner rectangle
+                new PIXI.Point(x + 10, y + 10), //top left of inner
+            ])
+
+            // create the graphics object for the hit area. this is not necessary, but it is helpful for debugging
+            const hitAreaGraphics = new PIXI.Graphics();
+            hitAreaGraphics.beginFill(0xFF0000, 0.0);
+            hitAreaGraphics.drawPolygon(invisibleHitArea);
+            hitAreaGraphics.endFill();
+
+            // add the hit area to the rectangle
+            rect.addChild(hitAreaGraphics);
+
+            rect.hitArea = invisibleHitArea;
+        }
+        return label;
```

Comment:
I essentially had to create a polygon that wraps around the label + rectangle, and assigned the graphics hitarea to be that.